simapp/simd/cmd/genaccounts.go

```diff
@@ -67,7 +67,7 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 				addr = info.GetAddress()
 			}
 
-			coins, err := sdk.ParseCoins(args[1])
```

Comment:
Should we make parseCoins a private function?

Reply:
Do you mean `sdk.ParseCoins()`? If so, why?

Reply:
From bez:

> All client-facing calls should call ParseCoinsNormalized

and ParseCoins seems to only be used in tests right now.

Reply:
OK agreed. Let's just replace `ParseCoins()` with `ParseCoinsNormalized()` then?

Reply:
Maybe we just change the implementation of `ParseCoins` then, if the old one is not needed?